i18n/fr/docusaurus-plugin-content-docs/current/full-node/Full-node-on-raspberry-pi.md

```diff
@@ -0,0 +1,161 @@
+```
+
+La commande ci-dessus vous demandera de confirmer que `/dev/disk2` est la carte SD et non votre disque dur, et vous demandera votre mot de passe car flasher une carte SD nécessite des privilèges d'administrateur.
+
+Une fois la commande ci-dessus terminée, vous pouvez insérer la carte SD dans votre Raspberry Pi et l'allumer.
+Il faut une poignée de minutes pour que le premier démarrage s'exécute complètement, et votre Raspberry Pi est prêt à être utilisé.
+Une fois que le noeud est prêt, vous pouvez y accéder en utilisant `alephium` comme nom d'utilisateur, et `installfest2021` comme mot de passe !
+
+```shell
+ssh alephium@alephium
+```
+
+Si l'hôte `alephium` est inconnu, vous devrez chercher l'adresse IP du noeud, très probablement sur la page/application de configuration de votre routeur.
+
+Et voilà, votre Raspberry Pi fonctionne sous Ubuntu 20.04 avec Docker, et est prêt à exécuter un noeud complet Alephium.
+
+🚀
+
+![Raspberry pi 4](media/pies.jpeg)
+
+## Comment faire fonctionner une instance d'Alephium full node sous Docker ?
+
+Cette deuxième section n'est pas spécifique à un Raspberry Pi, mais peut être généralisée à tout serveur/vm/ordinateur ayant un accès SSH.
+Nous allons exécuter la version la plus basique d'un noeud complet d'Alephium en utilisant Docker, et ensuite itérer pour rendre notre configuration plus pratique à utiliser.
+plus pratique pour travailler.
+
+Comme pré-requis de cette section, nous devons avoir un serveur avec un accès SSH, et plus précisément exécutant Ubuntu 20.04 ou plus récent.
+La section précédente explique comment faire cela avec un Raspberry Pi, mais une instance AWS EC2 ferait également l'affaire.
+
+### Connexion au serveur
+
+Cela devrait être une étape facile, en utilisant la commande `ssh`. Exécutez :
+
+```shell
+ssh alephium@alephium
+```
+
+### Installation de docker et docker-compose
+
+Installons rapidement docker et docker-compose, afin d'être prêts à faire fonctionner le noeud complet Alephium.
+
+Une fois connecté par ssh, exécutez les commandes suivantes :
+
+```shell
+sudo apt install -y docker.io docker-compose
+```
+
+Super, docker devrait fonctionner :
+
+```shell
+docker ps
+```
+
+### Exécuter le noeud complet
+
+Maintenant, nous pouvons exécuter le noeud complet, en une seule ligne, comme suit :
+
+```shell
+docker run -it --rm -p 12973:12973 --name alephium alephium/alephium:v1.2.6
+```
+
+### Docker-compose
+
+Docker-compose est une manière un peu plus pratique d'exécuter un conteneur, surtout si la commande commence à contenir
+des volumes, plus de ports, des variables d'environnement, etc...
+
+Donc, ci-dessous est la définition du service que vous pouvez mettre dans un fichier `docker-compose.yml`, et simplement appeler `docker-compose up -d` pour démarrer votre noeud complet à partir de cette définition.
+
+```yaml
+version : "3"
+services :
+  broker :
+    image : "alephium/alephium:v1.2.6"
+    restart : unless-stopped
+    ports :
```

Comment:
Here too

Reply:
fixed in https://github.com/alephium/wiki/pull/39/commits/aa98c0da5a79c88cbe3ac669aeaa6acb4a6c3f8f